src/templates/admin/core/article.html

```diff
@@ -22,6 +22,7 @@ <h2>{{ article.title }}</h2>
         </div>
     </div>
 
+    {% if not journal_settings.general.hide_review_data_pre_release or article.is_accepted or article.date_declined %}
```

Comment:
I think this logic means that checking the boolean will *always* hide reviews in the review stage, even if the editor marks a review as for author consumption. Am I reading it right? Do we want to make it so reviews released for author consumption show up during review, independent of the new setting's value?